src/ml_board/frontend/Dashboard/src/components/settings/Settings.tsx

```diff
@@ -15,14 +15,18 @@ export const defaultSocketConnectionUrlHelperText = "eg: http://127.0.0.1:5002";
 export const defaultRestApiUrlHelperText = "eg: http://127.0.0.1:5001";
 
 export interface FuncProps {
```

Comment:
are we using **FuncProps** elsewhere other than this file? if not then we shouldn't export it

Reply:
Yes, we r using it in ConfigPopup.tsx